RichClipboard.package/RichClipboard.class/instance/clipboardChanged.from..st

```diff
@@ -0,0 +1,3 @@
+events
+clipboardChanged: newClipboardText from: updateSource
```

Comment:
Perhaps `newClipping: newClipboardText from: updateSource` or `addedNewClipping: ...` would make the purpose of the method clearer? `clipboardChanged` seems a bit too vague.

Reply:
Geändert zu `clipboardChanged: newClipboardContent from: updateSource`